gcloud/pubsub/_gax.py

```diff
@@ -0,0 +1,178 @@
+# Copyright 2016 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""GAX wrapper for Pubsub API requests."""
+
+try:
+    # pylint: disable=no-name-in-module
+    from google.gax import CallOptions
+    from google.gax.errors import GaxError
+    from google.pubsub.v1.pubsub_pb2 import PubsubMessage
+    # pylint: enable=no-name-in-module
```

Comment:
But we fake the gRPC extras, just not the `gax` extras. See `_testing` our [use of it][1] in the config.

[1]: https://github.com/GoogleCloudPlatform/gcloud-python/blob/67693642144dee7d9c336abfe01885c8b54aae0f/tox.ini#L12-L13